pkgs/applications/misc/sigi/default.nix

```diff
@@ -2,14 +2,14 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "sigi";
-  version = "3.6.1";
+  version = "3.6.3";
 
   src = fetchCrate {
     inherit pname version;
-    sha256 = "sha256-UL4V/5XvqaqO4R2ievw379D/rzHf/ITgvG3BcSbMeTQ=";
+    sha256 = "sha256-JGQ9UbkS3Q1ohy6vtiUlPijuffH4Gb99cZCKreGqE/U=";
   };
 
-  cargoSha256 = "sha256-wzTUK4AvJmBK7LX7CLCAeAXLDxMJA/3qs/KT1+pMaoI=";
+  cargoSha256 = "sha256-W/ekk4tsYxG7FXzJW5i0Ii7nLgDHCSCjO3couN+/sMk=";
```

Comment:
https://nixos.org/manual/nixpkgs/stable/#rust

consider changing to `cargoHash` and `hash`

`buildRustPackage requires either the cargoSha256 or the cargoHash attribute which is computed over all crate sources of this package. cargoHash256 is used for traditional Nix SHA-256 hashes, such as the one in the example above. cargoHash should instead be used for [SRI](https://www.w3.org/TR/SRI/) hashes`

Reply:
I'm not quite sure I understand the benefits or preference from the docs... but I'm all for trying it out. Updated with this

Reply:
old attributes only supported single  of hash algorith eg sha256, which means when when calling overrideAttrs, you had to know what type of hash each derivation used

with  `hash` attribute, you can use SRI hash notation ( which you already using ) and then you can use different type of hash algorithm with same `hash` attribute

Reply:
Ahh, great!